int32[] stepperPostList
bool enableStepper